.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ['ubuntu-latest']
-        python-version: ['3', '3.8', '3.9']
+        python-version: ['3.8', '3.9', '3']
         include:
           - os: 'macos-latest'
             python-version: '3.8'  # oldest supported
```

Comment:
Aren't we going to new min version of 3.12 (ish)?

Reply:
Yes, but have we agreed on what release that will be?

Reply:
https://cylc.discourse.group/t/minimum-python-version/1191